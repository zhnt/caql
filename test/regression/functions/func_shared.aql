function shared() {
    print("Shared function executed")
}

function first() {
    print("First function calling shared")
    shared()
}

function second() {
    print("Second function calling shared")
    shared()
}

first()
second()


