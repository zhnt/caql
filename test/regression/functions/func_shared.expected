First function calling shared
Shared function executed
Second function calling shared
Shared function executed

